src/future/utils.js

```diff
@@ -0,0 +1,9 @@
+export function createRenderWith(_meta) {
```

Comment:
Do we really need this function if it only applies three fields to `Tag` (or whichever input)? I don't know if I see a lot of value for the abstraction over simply defining the fields.

Reply:
Probably not. The repeating `_meta={meta}` bugged me a bit and seemed like something that would easily get missed. And I didn't like the sort order rule of the linter changing from `_tag="div" _key="Field" _meta={meta}` to `_key="div" _meta={meta} _tag="div"`. But yeah, it's not that big of deal:tm:.